src/hooks/usePrefix.ts

```diff
@@ -3,10 +3,13 @@ import useConfig from "hooks/useConfig.ts"
 
 export class Prefix extends Path {
   www: Path
+  bin: Path
 
   constructor(prefix: Path) {
     super(prefix)
     this.www = prefix.join("tea.xyz/var/www")
+    // Maybe this should be usr/bin or tmp/bin?
+    this.bin = prefix.join("tea.xyz/bin")
```

Comment:
I'll change this after merge. I think we may go with `~/.cargo/bin` since it's the default and this works similarly to how `npx` and that work. Probs we should think about having all tea magic stuff cache inside `~/.tea` so users can clear it out. DUNNO YET!